akka-stream-tests/src/test/scala/akka/stream/scaladsl/CollectionSinkSpec.scala

```diff
@@ -36,7 +37,7 @@ class CollectionSinkSpec extends StreamSpec {
       "fail the future on abrupt termination" in {
         val mat = ActorMaterializer()
         val probe = TestPublisher.probe()
-        val future = Source.fromPublisher(probe).runWith(Sink.collection[Nothing, Seq[Nothing]])(mat)
+        val future = Source.fromPublisher(probe).runWith(Sink.collection[Unit, Seq[Unit]])(mat)
```

Comment:
I don't remember the exact error, but the type inference failed for `Nothing`. Since a collection of `Nothing` (i.e., one that can never have any elements) should be rare anyway I just update the test and didn't dive into it further.

Reply:
That's fine, mostly curious.